user_guide_src/source/testing/debugging.rst

```diff
@@ -172,3 +172,16 @@ The ``getVarData()`` method should return an array containing arrays of key/valu
 outer array's key is the name of the section on the Vars tab:
 
 .. literalinclude:: debugging/006.php
+
+.. _debug-toolbar-hot-reload:
+
+Hot Reloading
+=============
+
+The Debug Toolbar includes a feature called Hot Reloading that allows you to make changes to your application's code and have them automatically reloaded in the browser without having to refresh the page. This is a great time-saver during development.
+
+To enable Hot Reloading while you are developing, you can click the button on the left side of the toolbar that looks like a refresh icon. This will enable Hot Reloading for all pages until you disable it.
+
+Hot Reloading works by scanning the files within the ``app`` directory every second and looking for changes. If it finds any, it will send a message to the browser to reload the page. It does not scan any other directories, so if you are making changes to files outside of the ``app`` directory, you will need to manually refresh the page.
```

Comment:
Now we use ``**`` for file/directory path.

```suggestion
Hot Reloading works by scanning the files within the **app** directory every second and looking for changes. If it finds any, it will send a message to the browser to reload the page. It does not scan any other directories, so if you are making changes to files outside of the **app** directory, you will need to manually refresh the page.
```